docs/DeveloperGuide.md

```diff
@@ -510,7 +510,132 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**Use case: UC11 - View list of commands**
+**Use case: UC11 - Add client**
+
+**MSS**
+
+1.  User requests to add a client.
+2.  DevEnable adds the client to the client list.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The user makes an error in writing the request.
```

Comment:
I'm not sure if error handling should be considered under extensions, maybe you can clarify in the forum or otherwise

Reply:
Refer to [here](https://se-education.org/addressbook-level3/DeveloperGuide.html#use-cases) if you would like a better idea of what to do, the use cases I put up in the first iteration may not be up to par